src/node/rpc/member_frontend.h

```diff
@@ -10,18 +11,176 @@
 #include "node/secret_share.h"
 #include "node/share_manager.h"
 #include "node_interface.h"
+#include "tls/base64.h"
 #include "tls/key_pair.h"
 
 #include <charconv>
 #include <exception>
 #include <initializer_list>
 #include <map>
 #include <memory>
+#include <openenclave/attestation/verifier.h>
 #include <set>
 #include <sstream>
+#if defined(INSIDE_ENCLAVE) && !defined(VIRTUAL_ENCLAVE)
+#  include <openenclave/enclave.h>
+#else
+#  include <openenclave/host_verify.h>
+#endif
 
 namespace ccf
 {
+  class MemberTsr : public lua::TxScriptRunner
+  {
+    void setup_environment(
+      lua::Interpreter& li,
+      const std::optional<Script>& env_script) const override
+    {
+      auto l = li.get_state();
+      lua_register(l, "pem_to_der", lua_pem_to_der);
+      lua_register(
+        l, "verify_cert_and_get_claims", lua_verify_cert_and_get_claims);
+
+      TxScriptRunner::setup_environment(li, env_script);
+    }
+
+    static int lua_pem_to_der(lua_State* l)
+    {
+      std::string pem = get_var_string_from_args(l);
+      std::vector<uint8_t> der = tls::make_verifier(pem)->der_cert_data();
+      nlohmann::json json = der;
+      lua::push_raw(l, json);
+      return 1;
+    }
+
+#if 0
```

Comment:
```suggestion
#if USING_OE_11
```

would  be a little more greppable and descriptive :)